export const handleError = message => {
    console.log("Error")
    console.log(message);
}